docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -41,7 +41,6 @@ RUN python3 -m pip uninstall -y flax jax
 RUN python3 -m pip install --no-cache-dir torch-scatter -f https://data.pyg.org/whl/torch-$(python3 -c "from torch import version; print(version.__version__.split('+')[0])")+$CUDA.html
 RUN python3 -m pip install --no-cache-dir intel_extension_for_pytorch==$INTEL_TORCH_EXT+cpu -f https://software.intel.com/ipex-whl-stable
 
-RUN python3 -m pip install --no-cache-dir git+https://github.com/facebookresearch/detectron2.git pytesseract https://github.com/kpu/kenlm/archive/master.zip
```

Comment:
We should not remove the whole line, just the part `https://github.com/kpu/kenlm/archive/master.zip`

Reply:
Hi @raghavanone , once this comment is addressed, we are good to merge I think :-)